CMakeLists.txt

```diff
@@ -1588,7 +1588,19 @@ if(HEX_FILES_TO_MERGE)
 endif()
 
 if(EMU_PLATFORM)
-  include(${ZEPHYR_BASE}/cmake/emu/${EMU_PLATFORM}.cmake)
+  if(NOT EMU_PLATFORM_TARGET)
+    if(NOT EMU_PLATFORM_DEFAULT)
+      list(GET EMU_PLATFORM 0 EMU_PLATFORM_DEFAULT)
+      message(WARNING "EMU_PLATFORM_DEFAULT is not set! Defaulting to "${EMU_PLATFORM_DEFAULT})
```

Comment:
No warning here, users should be able to build without warnings if they just do `west build` or `cmake ...`

You can propose a status informing the user which emu platform has been selected, but i'm not convinced that even such a message should be printed at all.

There are boards where people are building for real hardware that still defines a `EMU_PLATFORM` and telling those users about the emu platform is just noise. 

Also it could be confusing to see such message in the cases where they actually intend to flash the board and will not be using an emulator.
https://docs.zephyrproject.org/latest/boards/arm/mps2_an385/doc/index.html
https://github.com/zephyrproject-rtos/zephyr/blob/3053a931a31b0f72a7b149edf8ed2a244f8273a9/boards/arm/mps2_an385/board.cmake#L3

Reply:
Thanks, I've removed the warning message.